publish.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+JOB_TYPE=${JOB_TYPE:-local}
```

Comment:
Nit: can we add a link to Prow docs (comment `# https://..`) where these env. variables are documented?